ui/libs/search/src/lib/state/common.ts

```diff
@@ -0,0 +1,20 @@
+import {HashMap, IHasId} from "@eosc-search-service/types";
+
+export const INITIAL_FILTER_OPTION_COUNT = 10;
```

Comment:
In the filter component, you used value 10, maybe this is the const that should be placed in there.